deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2

```diff
@@ -6,9 +6,18 @@
         <env name="DATABASE_PASSWORD" value="docker" />
         <env name="DATABASE_HOST" value="db" />
         <env name="INASAFE_REALTIME_BROKER_URL" value="amqp://guest:guest@{{ interpreters.inasafe_realtime.ipaddress }}:{{ docker_port_forward.inasafe_realtime.rabbitmq }}/" />
+
+        {% if inasafe_headless.use_prebuilt_docker %}
+
+        <env name="INASAFE_HEADLESS_BROKER_URL" value="amqp://guest:guest@rabbitmq:5672/" />
```

Comment:
ok, sorry for the late reply, I think I understand this now. You don't have to use if switch for this one. See the sample from my PR. the value can be taken from the default all.yml